resources/assets/js/components/FriendsList.vue

```diff
@@ -155,6 +166,27 @@
                 this.$http.patch('/api/conferences/' + this.conferenceSlug + '/' + this.key + '/' + friend.id, { met: met })
                     .then(() => { friend.met = met; });
             },
+
+            showDetails: function(friend) {
```

Comment:
I'd swap this out for its own Vue component with template and just pass the friend into it otherwise the variable concatenation and interpolating strings will get a bit messy, especially if we add to it in future.

You could then add `<friend-details :friend="friend"></friend-details>` into this component's template and pull it in. I'd maybe set the title to `@friend.username`, rather than an empty string, too.